ykrt/src/compile/jitc_yk/jit_ir/mod.rs

```diff
@@ -1296,6 +1296,22 @@ impl Operand {
             f(*iidx)
         }
     }
+
+    pub(crate) fn decopy(&self, m: &Module) -> Self {
```

Comment:
Needs a docstring. I think "If self is `Operand::Var` and references a `Copy`, recursively search until a non-`Copy` instruction is found." or similar will do.